### DISCUSSION TITLE
2023/11/24/%E6%97%A0%E9%A2%98%E4%B9%8B%E4%BA%8C/

### DISCUSSION THREAD

**Discussion by giscus** [cyclinbox/cyclinbox.github.io#6] [General]
# 2023/11/24/%E6%97%A0%E9%A2%98%E4%B9%8B%E4%BA%8C/

一篇没有标题的日记

https://wz.anoms.top/2023/11/24/%E6%97%A0%E9%A2%98%E4%B9%8B%E4%BA%8C/

**Comment by Hazelandinstant** [comment]
“首先是健康，其次是其次”   我最近的深刻感悟，生病浪费时间浪费钱，还耽误我科研，消耗我热情，导致我拖延，要不然，我也是能搞一搞zotero和ob的！

**Reply by cyclinbox** [reply]
抱抱，“首先是健康”不能再同意了。最近肺炎和流感太严重了，康复了也要好好休息保护自己哇~
